examples/00-basic-examples/EDB_Create_VIA.py

```diff
@@ -15,7 +15,10 @@
 import os
 from pyaedt import Edb
 from pyaedt.generic.general_methods import generate_unique_name
-aedb_path=os.path.join(r"C:\Temp", generate_unique_name("Edb_custom")+".aedb")
+if os.name == "posix":
+    aedb_path = os.path.join("/tmp", generate_unique_name("Edb_custom") + ".aedb")
+else:
+    aedb_path=os.path.join(r"C:\Temp", generate_unique_name("Edb_custom")+".aedb")
```

Comment:
Not necessary to change it now, but in the future you can use `/` instead of `\` to avoid having to use raw strings.